lms/djangoapps/oauth_dispatch/tests/test_views.py

```diff
@@ -318,3 +318,81 @@ def test_get_view_for_dop(self):
     def test_get_view_for_no_backend(self):
         view_object = views.AccessTokenView()
         self.assertRaises(KeyError, view_object.get_view_for_backend, None)
+
+
+class TestRevokeTokenView(_DispatchingViewTestCase):  # pylint: disable=abstract-method
+    """
+    Test class for RevokeTokenView
+    """
+
+    login_with_access_token_url = reverse("login_with_access_token")
+    revoke_token_url = reverse('revoke_token')
+    access_token_url = reverse('access_token')
+
+    def setUp(self):
+        super(TestRevokeTokenView, self).setUp()
+        response = self.client.post(self.access_token_url, self.access_token_post_body())
+        access_token_data = json.loads(response.content)
+        self.access_token = access_token_data['access_token']
+        self.refresh_token = access_token_data['refresh_token']
+
+    def access_token_post_body(self, refresh_token=None):
+        """
+        Returns a dictionary to be used as the body of the access_token POST request
+        """
+        if refresh_token:
+            return {
+                'client_id': self.dot_app.client_id,
+                'grant_type': 'refresh_token',
+                'refresh_token': refresh_token,
+            }
+        else:
+            return {
+                'client_id': self.dot_app.client_id,
+                'grant_type': 'password',
+                'username': self.user.username,
+                'password': 'test',
+            }
+
+    def revoke_token_post_body(self, token):
+        """
+        Returns a dictionary to be used as the body of the revoke_token POST request
+        """
+        return {
+            'client_id': self.dot_app.client_id,
+            'token': token,
+        }
+
+    def login_with_access_token(self):
+        """
+        Login with access token and return response
+        """
+        return self.client.post(
+            self.login_with_access_token_url,
+            HTTP_AUTHORIZATION="Bearer {0}".format(self.access_token)
+        )
+
+    def verify_revoke_token(self, token):
+        """
+        Verifies access of token before and after revoking
+        """
+        self.assertEqual(self.login_with_access_token().status_code, 204)
+
+        response = self.client.post(self.revoke_token_url, self.revoke_token_post_body(token))
+        self.assertEqual(response.status_code, 200)
+
+        self.assertEqual(self.login_with_access_token().status_code, 401)
+        response = self.client.post(self.access_token_url, self.access_token_post_body(self.refresh_token))
+        self.assertEqual(response.status_code, 401)
```

Comment:
Nit: Create a separate helper method for the last 2 lines so it's easier to understand what that logic is for.  Consider doing the same for L384 as well.  So in the end, the code would look like:

```
self._assert_access_token_invalidated()
self._assert_refresh_token_invalidated()
```